dom/src/ElementFinder.ts

```diff
@@ -46,3 +43,7 @@ export class ElementFinder {
       .concat(topNodeMatches);
   }
 }
+
+function toElArray(input: any): Array<Element> {
```

Comment:
Let's move this fellow up, before class ElementFinder, to follow the `x - n` rule.